plugins/admin.sma

```diff
@@ -792,7 +792,7 @@ accessUser(id, name[] = "")
 	
 	if (result & 1)
 	{
-		client_cmd(id, "echo ^"* %L^"", id, "INV_PAS")
+		console_print(id, "* %L, id, "INV_PAS")
```

Comment:
You should use engclient_print instead of console_print, because console_print doesn't work for connecting players.